src/App.css

```diff
@@ -16,6 +16,66 @@ main {
   bottom: 0;
 }
 
+.splash-container {
```

Comment:
## Status: ✔️ Project Approved ✔️ 
----

Hello @AmineHLub  🔥 

> # Excellent work on your project, it looks very professional and it's completed. Congrats on the good work!

![surprised](https://user-images.githubusercontent.com/60631456/122959988-c881fc00-d348-11eb-988a-a5aa782f4777.gif)

Now you can celebrate! 🥇 🚀 👍 💯 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️
`There's nothing left to say but time to merge! 😊`

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._